docs/commands/super.md

```diff
@@ -139,16 +139,16 @@ in the order appearing on the command line forming the input stream.
 |  Option   | Auto | Specification                            |
 |-----------|------|------------------------------------------|
 | `arrows`  |  yes | [Arrow IPC Stream Format](https://arrow.apache.org/docs/format/Columnar.html#ipc-streaming-format) |
-| `bsup`    |  yes | [Super Binary](../formats/bsup.md) |
-| `csup`    |  yes | [Super Columnar](../formats/csup.md) |
+| `bsup`    |  yes | [BSUP](../formats/bsup.md) |
+| `csup`    |  yes | [CSUP](../formats/csup.md) |
 | `csv`     |  yes | [Comma-Separated Values (RFC 4180)](https://www.rfc-editor.org/rfc/rfc4180.html) |
 | `json`    |  yes | [JSON (RFC 8259)](https://www.rfc-editor.org/rfc/rfc8259.html) |
 | `line`    |  no  | One string value per input line |
 | `parquet` |  yes | [Apache Parquet](https://github.com/apache/parquet-format) |
-| `sup`     |  yes | [Super JSON](../formats/sup.md) |
+| `sup`     |  yes | [SUP](../formats/sup.md) |
 | `tsv`     |  yes | [Tab-Separated Values](https://en.wikipedia.org/wiki/Tab-separated_values) |
 | `zeek`    |  yes | [Zeek Logs](https://docs.zeek.org/en/master/logs/index.html) |
-| `zjson`   |  yes | [Super JSON over JSON](../formats/zjson.md) |
+| `zjson`   |  yes | [Super over JSON (JSUP)](../formats/zjson.md) |
```

Comment:
Now that "Super JSON (JSUP)" is no longer a thing, I've heard rough consensus that what we'd been calling "ZJSON" could now take over the "JSUP" acronym and stand for "SUP over JSON". I started making that change as part of this PR but had to quickly confront the fact that there's ztests in the ZJSON format doc that specify the format via `-f zjson` and that's when I decided to back off and wait for a developer to first make the necessary code changes for how the format is referenced in the tooling itself. However, in the spirit of getting all mentions of the text "Super JSON" out of the docs I did make these presumptive name changes in the textual references in these format tables.